policyengine_us/variables/gov/states/nc/tax/income/nc_income_tax_before_refundable_credits.py

```diff
@@ -0,0 +1,15 @@
+from policyengine_us.model_api import *
+
+
+class nc_income_tax_before_refundable_credits(Variable):
```

Comment:
Does NC have any non-refundable credits? If so this should be `nc_income_tax_before_credits`

Reply:
NC has the child tax credit but no refudable credits -> will adjust